post-processor/vagrant-cloud/client.go

```diff
@@ -65,43 +65,49 @@ func encodeBody(obj interface{}) (io.Reader, error) {
 	return buf, nil
 }
 
-func (v VagrantCloudClient) Get(path string) (*http.Response, error) {
-	params := url.Values{}
-	params.Set("access_token", v.AccessToken)
-	reqUrl := fmt.Sprintf("%s/%s?%s", v.BaseURL, path, params.Encode())
+func (v *VagrantCloudClient) ValidateAuthentication() error {
+	resp, err := v.Get("authenticate")
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != 200 {
+		return fmt.Errorf("Invalid credentials: %s", resp.Status)
```

Comment:
Perhaps be more specific with "Invalid Vagrant Cloud credentials"?

However I leave this to your discretion.